src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -41,7 +41,10 @@ public abstract class WorkspaceModel : NotificationObject, ILocatable, IUndoRedo
         private int currentPasteOffset = 0;
         internal int CurrentPasteOffset
         {
-            get { return currentPasteOffset; }
+            get
+            {
+                return currentPasteOffset == 0 ? PASTE_OFFSET_STEP : currentPasteOffset;
```

Comment:
We've been correcting `const` values to use camel case. Can we make this `PasteOffsetStep`?